frontend/src/libs/components/logout/logout.tsx

```diff
@@ -0,0 +1,22 @@
+import { Button } from '~/libs/components/components.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  onLogout: () => void;
+};
+
+const Logout: React.FC<Properties> = ({ onLogout }: Properties) => {
+  return (
+    <Button
```

Comment:
But it can be just derived to `Button` component with custom `onClick` and `className` props